tools/BaseRunner.py

```diff
@@ -48,7 +48,7 @@ def __init__(
             self,
             name,
             executable=None,
-            supported_features={'preprocessing', 'parsing', 'simulation'}):
+            supported_features={'preprocessing', 'parsing', 'elaboration'}):
         """Base runner class constructor
         Arguments:
         name -- runner name.
```

Comment:
Probably the logic in get_mode needs to be updated too.

Reply:
Right, I simplified it a little bit there